docs/data/date-pickers/date-range-calendar/date-range-calendar.md

```diff
@@ -28,11 +28,16 @@ The component can be disabled or read-only.
 
 ## Customization
 
-### Render 1 to 3 months
+### Choose the months to render
 
 You can render up to 3 months at the same time using the `calendars` prop:
 
-{{"demo": "DateRangeCalendarCalendarProp.js"}}
+{{"demo": "DateRangeCalendarCalendarsProp.js"}}
+
+You can render the month in the backward direction using the `currentMonthCalendarPosition` prop.
```

Comment:
```suggestion
You can control the position the current month is rendered in using the `currentMonthCalendarPosition` prop.
```

Reply:
maybe "choose", or "set" instead of  "control" since there is no `onCurrentMonthCalendarPositionChange` callback

Reply:
Good point. 👍